src/frontend/src/components/DownloadMetadataTemplate/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { CSVLink } from "react-csv";
+
+const TSV_SEPARATOR = "\t";
+
+interface Props {
+  rows: string[][];
+  headers: string[];
+  children: React.ReactNode;
+  className?: string;
+}
+
+const SampleUploadDownloadTemplate = ({
+  children,
+  rows,
+  headers,
+  className,
+}: Props): JSX.Element => {
+  return (
+    <CSVLink
+      className={className}
+      data={rows}
+      headers={headers}
+      filename="metadata_template.tsv"
+      separator={TSV_SEPARATOR}
+      data-test-id="download-tsv-link"
+    >
+      {children}
+    </CSVLink>
+  );
+};
+
+interface SampleEditTsvProps extends Props {
+  instructions: string[][];
+}
+
+const SampleEditTsvTemplateDownload = ({
+  children,
+  rows,
+  headers,
+  instructions,
+  className,
+}: SampleEditTsvProps): JSX.Element => {
+  return (
+    <CSVLink
+      className={className}
+      filename="metadata_template.tsv"
+      separator={TSV_SEPARATOR}
+      data={[...instructions, [...headers], ...rows]}
```

Comment:
```suggestion
      data={[...instructions, headers, ...rows]}
```

I think the above suggestion is correct. If headers is an array, and then you put it inside an array only to spread it then you just end up with the original array again, right?